docs/cloud/aws/budget-actions/budget-actions.md

```diff
@@ -19,7 +19,7 @@ You can attach budget actions to an alert for either a cost budget or a usage bu
 5.	Choose ‘Budget scope’ and select the services you would like in scope. For this example we will choose 'All AWS Services' and choose ‘Next’
     ![](https://raw.githubusercontent.com/ResearchComputing/Documentation/master/docs/cloud/aws/budget-actions/images/3.png)
```

Comment:
This is a strange thing. Can we please remove this link and use a proper Markdown reference. Could you please do this for any other items that are like this?

Reply:
Isn't this the correct markdown for images?

Reply:
Markdown syntax is fine. What I am referencing is the use of the GitHub link. This is an image that is in our docs, so it should be internally referenced.

Reply:
436c85e